src/client/datascience/interactive-common/interactiveBase.ts

```diff
@@ -321,14 +318,6 @@ export abstract class InteractiveBase extends WebviewPanelHost<IInteractiveWindo
                 this.handleMessage(message, payload, this.requestVariables);
```

Comment:
What is this file still used for? Seems like it can go away?

Reply:
Please see comments, I'm going to remove the code from extension in a separate PR.
Else this PR gets way too big & too difficult, this way I can just remove the UI code & the bundle & later focus on extension & then focus on cleaning up everything (3 prs).
Primarily because I find that easier, else I'd still be working on cleaning up extension code (this PR gets the UI out)